site/content/docs/user/known-issues.md

```diff
@@ -209,19 +202,12 @@ docker image prune
 
 You can verify the issue by exporting the logs (`kind export logs`) and looking
 at the kubelet logs, which may have something like the following:
-```
+
+```txt
 Dec 07 00:37:53 kind-1-control-plane kubelet[688]: I1207 00:37:53.229561     688 eviction_manager.go:340] eviction manager: must evict pod(s) to reclaim ephemeral-storage
 Dec 07 00:37:53 kind-1-control-plane kubelet[688]: E1207 00:37:53.229638     688 eviction_manager.go:351] eviction manager: eviction thresholds have been met, but no pods are active to evict
 ```
-## Failure to Create Cluster with Cgroups v2
```

Comment:
I guess we can refer back to this in git / ask people to please upgrade. It's surprising how often we still get issues from really old releases of both kind and Kubernetes